lib/common.sh

```diff
@@ -11,25 +11,38 @@ DAPP_LIB=${DAPP_LIB:-$BIN_DIR/contracts}
 
 export OUT_DIR=${OUT_DIR:-$PWD/out}
 
-CONFIG_FILE="$CONFIG_DIR/$CONFIG_STEP.json"
+CONFIG_FILE="${OUT_DIR}/config.json"
+# Only executes if called from the initial script
+if [[ "$CONFIG_STEP" = "$1" ]]; then
+    # Clean out directory
+    rm -rf "$OUT_DIR"
+    mkdir "$OUT_DIR"
```

Comment:
Can we make them one line then, with `&&`? I think it looks cleaner.

Reply:
Agreed

Reply:
Done.